app/views/wikis/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="container">
   <% @wikis.each do |wiki| %>
-    <strong>Title: </strong><%= wiki.title %><br />
+    <strong>Title: </strong><%= link_to wiki.title, wiki %><br />
     <strong>Body: </strong><%= wiki.body %><br />
     <strong>Created by: </strong><%= link_to wiki.user.email, wiki %><br />
```

Comment:
I forgot to mention this when I opened the pull request but what is the best way to setup the user email link to route to that specific users profile page? My route for the user#show is `/users/:id(.:format)` which tells me that I have to pass the URL the users ID so I tried `user_path(@user)` with no luck. What am I missing here? It seems like a easy addition but the ways I'm "used" to doing it just aren't working. @bjblock